src/main/java/com/lms/learning_management_system/entities/UserEntity.java

```diff
@@ -0,0 +1,21 @@
+package com.lms.learning_management_system.entities;
+
+import jakarta.persistence.*;
+import lombok.Data;
+import java.util.UUID;
+
+@Entity
+@Data
```

Comment:
@Data create a constructor with all NonNull and final fields, because you have none of these your code produces a NoArgsConstructor, it's good to add @AllArgsConstructor and @NoArgsConstructor as it is right now, if you change a field to final or annotate a fild with NonNull this will change the constructor as well without you noticing.